src/ext/NetFx/ca/netfxca.vcxproj

```diff
@@ -65,7 +65,16 @@
     <PackageReference Include="WixToolset.WcaUtil" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" PrivateAssets="All" />
     <PackageReference Include="GitInfo" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.NET.Tools.NETCoreCheck.x86" GeneratePathProperty="true" />
+    <PackageReference Include="Microsoft.NET.Tools.NETCoreCheck.x64" GeneratePathProperty="true" />
+    <PackageReference Include="Microsoft.NET.Tools.NETCoreCheck.arm64" GeneratePathProperty="true" />
   </ItemGroup>
 
+  <Target Name="CopyNetCoreCheck" AfterTargets="Build">
```

Comment:
I don't think these files should be copied anywhere. You should be able to add bind paths in the .wixproj for these locations and reference them that way. If you were using the hostfxr.dll as an example from my PR, then that was different because I needed it in the output directory to run the .exe from VS while developing it.